messages/run-dfa.js

```diff
@@ -6,17 +6,17 @@ module.exports = {
 	"flags": {
 		"formatDescription": "format of results",
 		"formatDescriptionLong": "Specifies output format with results written directly to the console.",
-		"ignoreparseerrorsDescription": "ignore compilation failures in scanned files",
+		"ignoreparseerrorsDescription": "ignore compilation failures in scanned files (default: false). Alternatively, set value using environment variable `SFGE_IGNORE_PARSE_ERRORS`",
 		"ignoreparseerrorsDescriptionLong": "ignore compilation failures in scanned files. Inherits value from SFGE_IGNORE_PARSE_ERRORS env-var if set.",
-		"normalizesevDescription": "A normalized severity 1 (high), 2 (moderate), and 3 (low) is returned in addition to the engine specific severity",
+		"normalizesevDescription": "return normalized severity in addition to the engine specific severity",
 		"normalizesevDescriptionLong": "A normalized severity 1 (high), 2 (moderate), and 3 (low) is returned in addition to the engine specific severity. For the html option, the normalized severity is displayed instead of the engine severity",
 		"outfileDescription": "location of output file",
 		"outfileDescriptionLong": "Write output to a file.",
-		"projectdirDescription": "Root project directory",
+		"projectdirDescription": "directory where the target location resides in or the context of the remaining files in the target's project",
```

Comment:
I don't like this, because it implies that you could pass in `force-app/main/default/classes`, when we really want `force-app/main/default`.

Reply:
How about this:
```
directory where the target location resides in along with the remaining class files or the project's root
```